src/middlewares/noAuth.middleware.js

```diff
@@ -0,0 +1,23 @@
+const { ApiError } = require('../exceptions/api.error');
+const jwtService = require('../services/jwt.service');
+
+const noAuthMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (authorization || token) {
```

Comment:
The condition here is incorrect. The logic should be adjusted to check if a token is present and valid, and only then throw a forbidden error. Currently, it throws an error even if the token is not present, which is not the intended behavior.